_layouts/people.html

```diff
@@ -142,15 +132,15 @@ <h2>Details</h2>
                 {% endif %}
             {% endfor %}
         {% endif %}
-        {% if c.hosts %}
-            {% for h in c.hosts %}
-                {% capture ols-3-hosts %}{{ ols-3-hosts}} {{ h }}{% endcapture %}
+        {% if c.facilitators %}
+            {% for h in c.facilitators %}
+                {% capture ols-3-facilitators %}{{ ols-3-facilitators}} {{ h }}{% endcapture %}
```

Comment:
You can remove this part then

Reply:
Line 138?

Reply:
Thank you. 👀

Reply:
Nevermind, @bebatut. I realised I had said something dumb and then "erased" it. 😄